routes/mountains.js

```diff
@@ -0,0 +1,191 @@
+const db = require('../db');
+const express = require('express');
+app = express.Router();
```

Comment:
please always initialize variables with `const` or `let`
also, typically the variable `router` is used in express router files because it's more descriptive than `app`